### ISSUE TITLE
Matrix.org comparison not updated

### ISSUE THREAD

**Issue by nordurljosahvida** [berty/www.berty.tech#103]
This is a suggestion and a request for comment for [the matrix comparison](https://github.com/berty/www.berty.tech/blob/master/bertytech/content/compare-table-apps/matrix.md)

1. is there a specific reason I'm not aware of for which the `open source` value was left blank?
2. based on these [latest developments](https://matrix.org/blog/2020/06/02/introducing-p-2-p-matrix) I think we should at some point consider changing their `distributed` tab to `partially` or wait for the implementation to be final to change it to `true`?
3. consider e2ee is now default in 2-participant rooms, but yes this does not change the status to `true`
4. why is the `aosp` tab void? does riot / riotx android client not work without GCM?

I think this is important as IMO Matrix [along with signal] is the real leader on the bleeding edge out there, and is investing a lot in P2P, so it'd be interesting to cross-reference approaches and advantages/disadvantages with them. Shoutout to Jami and status.im of course.

Thanks!

**Comment by pierreboc** [comment]
Hi @nordurljosahvida 

1) Open source can be filled 'True' (https://matrix.org/faq/#what-do-you-mean-by-open%3F: Matrix is also open source, meaning that we have released the source code of the reference servers, clients and services to the public domain under the Apache Licence v2, to encourage anyone and everyone to run their own servers and clients, and enhance them and contribute their enhancements as they see fit.)
2) Considering this post, we can switch Distributed from 'False' to 'Partially'.
3) Since 2020-05-06, Matrix is is end-to-end encrypted by default for private conversations. So, E2EE can be switched to true.
4) It is precisely when we are not sure of the answer that we leave the box blank. There is a button to create a PR if someone wants to make a change to the table with if possible a link to a source/screenshot.

I just did a PR to update accordingly. Thanks for your input.
